### PR TITLE
Transit/Switzerland: Added 'unknown' for missing platform values (fixes #1835)

### LINKED ISSUE #1835: Transit Switzerland: API not always providing platform, leads to bad result
https://duckduckgo.com/?q=next+train+to+geneva+from+paris&ia=swisstrains

The footer in the IA shows "Platform " with no number because the API response doesn't contain one. We should check to make sure we have a platform, or show that one has not been assigned yet (I'm assuming that's the case?)

https://duckduckgo.com/js/spice/transit/switzerland/paris/geneva

/cc @tagawa @mattr555

### PR DESCRIPTION
**Pull request by tagawa**

### CODE REVIEW

File: share/spice/transit/switzerland/transit_switzerland.js [hunk 1]
```diff
@@ -32,7 +32,7 @@
                     arrival_time: format_time(item.to.arrival),
                     departure_time: format_time(item.from.departure),
                     name: item.sections[0].journey.name,
-                    platform: item.from.platform,
+                    platform: (item.from.platform) ? item.from.platform : 'unknown',
```
Comment (moollaza) [inline share/spice/transit/switzerland/transit_switzerland.js]: I think an `||` would be a little simpler?

```js
platform: item.from.platform || 'unknown';
```

Comment (tagawa) [inline share/spice/transit/switzerland/transit_switzerland.js]: What was I thinking?! Thanks - updated.

**Comment by mattr555** [comment]
@tagawa Thank you! In order to be [consistent](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/transit/njt/train_item.handlebars#L17) we may want to change it to "N/A" and gray the text to `#aaa`.

BTW @iambibhas also presents a fix to this on #1784. This solution is probably better, though, as hiding the platform may cause some tiles to be taller than others.

**Comment by tagawa** [comment]
Thanks @mattr555, good idea - done.

**Comment by mattr555** [comment]
screenshot - 05142015 - 09 34 58 pm
:+1: Thanks!

**Comment by iambibhas** [comment]
This looks better. :)

**Pull request merged**